.changes/v4.0.0/1346-improvement.md

```diff
@@ -0,0 +1,2 @@
+* Improve resource `vcd_vm_internal_disk` documentation with HCL sample to  achieve correct order of
```

Comment:
Name of file should be `.changes/v4.0.0/1346-improvements.md`

Reply:
Fixed.